api/jobs.go

```diff
@@ -117,10 +117,15 @@ func (api *API) GetJobHandler(ctx context.Context) http.HandlerFunc {
 func (api *API) GetJobsHandler(w http.ResponseWriter, req *http.Request) {
 	ctx := req.Context()
 	log.Info(ctx, "Entering handler function, which calls GetJobs and returns a list of existing Job resources held in the JobStore.")
-	offsetParameter := req.URL.Query().Get("offset")
-	limitParameter := req.URL.Query().Get("limit")
+	offsetParam := req.URL.Query().Get("offset")
+	limitParam := req.URL.Query().Get("limit")
 
-	jobs, err := api.dataStore.GetJobs(ctx, offsetParameter, limitParameter)
+	offset, limit, setUpPaginationFailed := api.setUpPagination(w, offsetParam, limitParam, ctx)
+	if setUpPaginationFailed {
+		return
+	}
+
+	jobs, err := api.dataStore.GetJobs(ctx, offset, limit)
 	if err != nil {
 		switch {
 		case badRequest[err]:
```

Comment:
I believe this case statement is pointless now and could go (together with the `badRequest` map)

Reply:
yes - deleted both